en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,474 @@
+
+Similarly, you can expand and try out the **GET** and **DELETE** methods.
+
+
+### Step 1.4: Publish the REST API
+
+In the previous step, you tested the `readingList` REST API and found that it works as expected. Therefore, let's publish it and make it available for subscriptions.
+
+#### Step 1.4.1: Update the CORS configuration
+
+The application developer in this scenario calls the API from a different domain (i.e., Vercel). By default, web browsers block these calls for security reasons. To enable the application to call the API, follow the steps given below:
+
+1. In the left pane, click the **Manage** icon.
+
+2. Click **Settings**.
+
+3. Under **API Settings** click **Edit**.
+
+4. Toggle the **CORS Configuration** switch to enable the CORS configuration.
+
+5. Select the **Access Control Allow Credentials** checkbox.
+
+6. In the **Apply to Development** pane that opens on the right of the page, enter a meaningful message. Then click **Apply**.
+
+7. Click **Save**.
+
+#### Step 1.4.2: Publish the REST API
+
+To publish the REST API, follow the steps given below:
+
+1. In the **Manage** tab, click **Lifecycle**.
+
+2. Click **Publish** to publish the REST API to the Developer Portal.
+
+3. To access the Developer Portal, click **Go to DevPortal**.
+
+    The readingList REST API will open in the Developer Portal.
+
+## Step 2: Consume the REST API
+
+You have published the readingList REST API to the Developer Portal where application developers can find it and subscribe their applications to it.
+
+In the previous steps, you played the role of a REST API developer and developed a REST API. In this step, you will play the role of a consumer of this REST API.
+
+To consume the  `readingList` REST API, let's create an application, subscribe it to the REST API, and send a request to it.
+
+### Step 2.1: Create an application
+
+First, let's create an application by following the steps given below:
+
+1. In the top menu of the Developer Portal, click **Applications**.
+
+2. Click **Create**.
+
+3. Enter `readingListApp` as the name of the application and click **Create**.
+
+    Your Application will open on a separate page.
+
+### Step 2.2: Subscribe to your application to the API
+
+To subscribe your application to the API, follow the steps given below:
+
+1. In the left navigation menu, click **Subscriptions**.
+
+2. Click **Add APIs**.
+
+3. Find your REST API and click **Add** for it.
+
+Now your application has subscribed to the `readingList` REST API.
+
+### Step 2.3: Generate a token for your application
+
+To generate a security token that your application can use to invoke the REST API, follow the steps given below:
+
+1. In the left navigation menu, click **OAuth 2.0 Tokens**.
+
+2. Click **Generate Credentials**.
+
+3. Click **Generate Test Token**.
+
+4. Copy the test token that appears by clicking the icon for copying and save it for later use.
```

Comment:
We don't need to specify the copy icon do we? We shouldn't have to explain that level of detail IMO.